instrumentation/kafka/src/test/java/brave/kafka/RecordTracingTest.java

```diff
@@ -0,0 +1,101 @@
+package brave.kafka;
+
+import brave.Span;
+import brave.Tracing;
+import brave.propagation.TraceContext;
+import brave.sampler.Sampler;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import zipkin.internal.Util;
+
+import java.io.IOException;
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+
+public class RecordTracingTest {
+    private static final String TRACE_ID = "463ac35c9f6413ad";
+    private static final String PARENT_ID = "463ac35c9f6413ab";
+    private static final String SPAN_ID = "48485a3953bb6124";
+    private static final String SAMPLED = "1";
+
+    private static final String TEST_TOPIC = "myTopic";
+    private static final String TEST_KEY = "foo";
+    private static final String TEST_VALUE = "bar";
+
+    private RecordTracing recordTracing;
+    private ConsumerRecord<String, String> fakeRecord;
+
+    private ConcurrentLinkedDeque<zipkin.Span> spans = new ConcurrentLinkedDeque<>();
```

Comment:
unless concurrent, don't use concurrent. it is distracting as I will start thinking there is concurrency going on :)